### PR TITLE
[Fix] wagmi ref guide htmldocs for using Auth v1.0.8 release

### PR DESCRIPTION
**Pull request by shaloo**
Changes

1.) Upgraded to v1.0.8 auth in pkg.json and now htmldocs are generated. There is one warning about `[warning] LoginType, defined in ./src/connector.ts, is referenced by ArcanaConnector.setLogin.setLogin.val but not included in the documentation.`.

2.) Updated Readme similar to other public repos.

3.) Moved usage information from readme to usage.md similar to Arcana Auth repo.

Preview via mybranch (will move tomain post merge) https://dev-authwagmisdk-ref-guide.netlify.app/

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,76 +1,78 @@
...
 ```
 
-## Configuration
+# 📋 Prerequisites
 
-## Add Chains
+Before you can start using the Arcana Auth SDK and Arcana Auth-Wagmi SDK in dApps, you need to register your dApp using [Arcana Developer Dashboard](https://dashboard.arcana.network/).
 
-Configure the chains that will show up in the Arcana wallet within your app's context. In the example below, the chains configured are: Optimism, Polygon and Ethereum Mainnet.
+A unique **App Address** will be assigned to your dApp and you need the same to initialize the Arcana Auth SDK.
```
Comment (anivenu) [inline readme.md:64]:
```suggestion
A unique **Client ID** will be assigned to your dApp and you need the same to initialize the Arcana Auth SDK.
```

File: readme.md [hunk 2]
```diff
@@ -1,76 +1,78 @@
...
 
-```ts
-import { mainnet, optimism, polygon } from '@wagmi/core/chains'
-import { ArcanaConnector } from "@arcana/auth-wagmi"
-import { AuthProvider } from "@arcana/auth"
+# 📚 Documentation
 
-const auth = new AuthProvider(`${arcana_client_id}`)
+Check out [Arcana Network documentation](https://docs.arcana.network/) and Auth-Wagmi SDK [Usage Guide](./usage.md].  Also see [Auth-Wagmi Quick Start Guide](https://docs.arcana.network/quick-start/wagmi-quick-start.html).
```
Comment (anivenu) [inline readme.md:68]:
```suggestion
Check out [Arcana Network documentation](https://docs.arcana.network/) and Auth-Wagmi SDK [Usage Guide](./usage.md).  Also see [Auth-Wagmi Quick Start Guide](https://docs.arcana.network/quick-start/wagmi-quick-start.html).
```

**Review by anivenu — APPROVED** [review]
Have a few minor comments. Approving assuming all the code snippets work and have been verified with latest auth SDK. @shaloo

File: readme.md [hunk 3]
```diff
@@ -1,76 +1,78 @@
...
-const connector = new ArcanaConnector({
-  chains: [mainnet, optimism, polygon],
-  options: {
-    auth,
-  },
-})
-```
+# 💡 Support
+
+For any support or integration-related queries, contact [Arcana support team](mailto:support@arcana.network).
```
[the remark this answers is not in the document]
Comment (shaloo) [reply readme.md:72]: Updating

**Pull request merged**